packages/babel-types/src/definitions/flow.js

```diff
@@ -250,15 +250,15 @@ defineType("NumberTypeAnnotation", {
 });
 
 defineType("ObjectTypeAnnotation", {
-  visitor: ["properties", "indexers", "callProperties"],
+  visitor: ["properties", "indexers", "callProperties", "exact"],
   aliases: ["Flow", "FlowType"],
   fields: {
     properties: validate(
       arrayOfType(["ObjectTypeProperty", "ObjectTypeSpreadProperty"]),
     ),
     indexers: validateOptional(arrayOfType("ObjectTypeIndexer")),
     callProperties: validateOptional(arrayOfType("ObjectTypeCallProperty")),
-    exact: validate(assertValueType("boolean")),
+    exact: validateOptional(assertValueType("boolean")),
```

Comment:
Why is this optional? It should be required but defaulting to `false` (I don't remember if it is actually possible to set default properties value)